hubmap_api_py_client/external.py

```diff
@@ -163,12 +163,83 @@ def _add_method(output_type, ResultsSetSubclass, doc):
 
 
 for output_type, doc in {
-    'cell': 'TODO: document cell',
-    'organ': 'TODO: document organ',
-    'gene': 'TODO: document gene',
-    'cluster': 'TODO: document cluster',
-    'dataset': 'TODO: document dataset',
-    'protein': 'TODO: document protein'
+    'cell':
+    '''
+    Select a set of cells. If no params are provided, selects the set of all cells.
+    Otherwise, selects a set of cells filtered based on parameters supplied.
+        where->str The type of entity being supplied as input to query, must be one of
```

Comment:
Is this a standard format for documenting arguments? There are several out there that tool chains will support. I'd suggest [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), unless there's a different standard you would prefer.

Reply:
I was primarily focused on content for this draft.  I'm happy to move to Google style.